protocols/relay/src/behaviour.rs

```diff
@@ -151,6 +153,8 @@ impl Default for Config {
             reservation_duration: Duration::from_secs(60 * 60),
             reservation_rate_limiters,
 
+            substream_timeout: Duration::from_secs(10),
+            max_concurrent_streams: 100,
```

Comment:
Is there some rational for choosing `100` here? This is the max per connection, correct? Why have two orders of magnitude more streams than the allowed number of circuits per peer? `10` seems more reasonable to me. Would you agree?

```suggestion
            max_concurrent_streams: 10,
```

Reply:
I copied `100` from Tomas's PR, no deep thinking ))